api/v1beta1/types.go

```diff
@@ -371,4 +371,12 @@ type LoadBalancer struct {
 	// required for the Load Balancer, if not defined the first configured subnet will be
 	// used.
 	Subnet *string `json:"subnet,omitempty"`
+
+	// AllowGlobalAccess. If set to true, clients can access the internal
+	// passthrough Network Load Balancers, the regional internal Application Load
+	// Balancer, and the regional internal proxy Network Load Balancer from all
+	// regions. If false, only allows access from the local region the load
+	// balancer is located at. Note that for INTERNAL_MANAGED forwarding rules,
+	// this field cannot be changed after the forwarding rule is created.
+	AllowGlobalAccess bool `json:"allowGlobalAccess,omitempty"`
```

Comment:
It is hard to tell based on the comments in the file, but I feel like this field should exist somewhere else or this struct name needs to change (or the comments should be altered slightly). The struct appears to only be used for Internal Load Balancers but the way that the comments are worded make this struct seem like it should be for external and internal Load Balancers. Allow global access only appears to be related to internal LBs, so something isn't matched in the description(s) here. 

nit: not necessarily a problem with this PR but maybe we should address it.